### PR TITLE
CODENVY-171: fix integration tests; decouple from Codenvy 3.x subscription and user's account

### PR DESCRIPTION
**Pull request by dmytro-ndp**
Changes:
1) fix integration tests of login and trial subscription to use Codenvy SaaS server https://codenvy-stg.com instead of http://nightly.codenvy-stg.com; 
2) decouple _IM CLI_, _Installation manager service_ and _Update Server_ from work around _Codenvy 3.x_ trial subscription and user's account.

@tolusha please, review this pull request.

Signed-off-by: Dmytro Nochevnov <dnochevnov@codenvy.com>

### CODE REVIEW

File: installation-manager-core/src/main/java/com/codenvy/im/facade/InstallationManagerFacade.java [hunk 1]
```diff
@@ -274,25 +230,22 @@ public UpdateArtifactInfo apply(Map.Entry<Artifact, Version> entry) {
...
-                try {
-                    if (downloadManager.getDownloadedVersions(artifact).containsKey(version)) {
-                        info.setStatus(UpdateArtifactInfo.Status.DOWNLOADED);
-                    } else {
-                        info.setStatus(UpdateArtifactInfo.Status.AVAILABLE_TO_DOWNLOAD);
-                    }
-                } catch (IOException e) {
-                    // simply ignore
-                }
+        infos.addAll(FluentIterable.from(allUpdates).transform(entry -> {
```
Comment (sleshchenko) [inline installation-manager-core/src/main/java/com/codenvy/im/facade/InstallationManagerFacade.java]: Can you use stream API here?

**Comment by tolusha** [comment]
+1

File: installation-manager-core/src/main/java/com/codenvy/im/facade/InstallationManagerFacade.java (hunk 1 above)
Comment (dmytro-ndp) [inline installation-manager-core/src/main/java/com/codenvy/im/facade/InstallationManagerFacade.java]: Yes, we can. But such kind of fix isn't the goal of the issue.

Comment (sleshchenko) [inline installation-manager-core/src/main/java/com/codenvy/im/facade/InstallationManagerFacade.java]: I think it is not hard to improve this code with stream API usage. Moreover you've already changed FluentIterable to Steam API in another place of this pull request https://github.com/codenvy/cdec/pull/26/files#diff-4c47e8f71dffba7574f409ab4b52444dL247
BTW Up to you

**Pull request merged**